src/Contexts/Shared/domain/CommandBus.ts

```diff
@@ -0,0 +1,5 @@
+import { Command } from './Command';
+
+export interface CommandBus {
+  dispatch(command: Command): Promise<void>;
+}
```

Comment:
What do you think about enabling the [a TSLint `eofline` rule](https://palantir.github.io/tslint/rules/eofline/)?

[Right now it's disabled](https://github.com/CodelyTV/typescript-ddd-skeleton/blob/master/tslint.json#L7).

Also, we should add the `tslint` execution to [the `npm test`](https://github.com/CodelyTV/typescript-ddd-skeleton/blob/master/package.json#L15) or some other task to be executed in [the project GitHub Workflow](https://github.com/CodelyTV/typescript-ddd-skeleton/blob/master/.github/workflows/nodejs.yml#L24-L28).

Do you agree? Am I missing something?

Reply:
Totally agree, I'm going to add it